examples/C++/HelloWorldExampleDS/HelloWorldServer.cpp

```diff
@@ -0,0 +1,102 @@
+// Copyright 2019 Proyectos y Sistemas de Mantenimiento SL (eProsima).
```

Comment:
```suggestion
// Copyright 2020 Proyectos y Sistemas de Mantenimiento SL (eProsima).
```

Reply:
HelloWorldExampleDS does not belong (yet) into fastdds.